src/modal-checkout/index.js

```diff
@@ -391,6 +398,15 @@ domReady(
 						} );
 						$form.on( 'submit', handleFormSubmit );
 					} else {
+						if ( $form.find( '[name="is_validation_only"]' ) ) {
+							$form.find( '[name="is_validation_only"]' ).remove();
+						}
```

Comment:
Nonblocking nitpick: Maybe we can just create a new variable for the first call to `find` here so we don't need to run this twice.

```suggestion
                        const validationOnlyField = $form.find( '[name="is_validation_only"]' );
                        if ( validationOnlyField.length ) {
							$validationOnlyField.remove();
						}

```

Reply:
Great idea!